anthos-bm-edge-deployment/README.md

```diff
@@ -62,10 +62,10 @@ The following quick start guide will take approximately **55-60 minutes** to com
 
 ```sh
 # once you have forked this repository, clone it to your local machine
-git clone https://github.com/<YOUR_GITHUB_USERNAME>/point-of-sale
+git clone https://github.com/<YOUR_GITHUB_USERNAME>/anthos-samples
 
 # move into the root of the infrastructure setup directory
-cd point-of-sale/anthos-baremetal-edge-deployment
+cd anthos-samples/anthos-bm-edge-deployment
```

Comment:
What's the context behind changing the name? Is "anthos bm" well understood by the community, or would it be better to keep the longform?

Reply:
The reduction from `baremetal` to `bm` is to make it consistent with the rest of the directories in this repo
- `anthos-bm-gcp-terraform`
- `anthos-bm-openstack-terraform`

Reply:
Gotcha, thank you.